tests/default_test.go

```diff
@@ -0,0 +1,61 @@
+package test
+
+import (
+	_ "WeatherAPI/routers"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/astaxie/beego"
+	. "github.com/smartystreets/goconvey/convey"
+)
+
+func init() {
+	_, file, _, _ := runtime.Caller(1)
+	log.Print(file)
+	// apppath, _ := filepath.Abs(filepath.Dir(filepath.Join(file, ".."+string(filepath.Separator))))
+	apppath := "/mnt/c/Users/carlosalberto.rojas/Desktop/go/src/WeatherAPI/"

```

Comment:
@carojaspy why aren't you using relative paths?

Reply:
Because beegon couldn't find the path to my tests, so to don't be stopped i used the absolute path

Reply:
@carojaspy, please try to use a relative path. If I tried to execute the code it will fail.

Reply:
Solved on this commit: 05579e04c696949f9f6726a45fbb6f9b113e1e0c